src/docfx/lib/schema/JsonSchemaValidator.cs

```diff
@@ -214,10 +215,26 @@ private void ValidateString(JsonSchema schema, string name, JValue scalar, strin
 
             switch (schema.Format)
             {
+                case JsonSchemaStringFormat.None:
+                    break;
+
                 case JsonSchemaStringFormat.DateTime:
-                    if (!DateTime.TryParse(str, out var _))
+                    if (!DateTime.TryParse(str, CultureInfo.InvariantCulture, DateTimeStyles.None, out var _))
                         errors.Add((name, Errors.FormatInvalid(JsonUtility.GetSourceInfo(scalar), str, JsonSchemaStringFormat.DateTime)));
                     break;
+
+                case JsonSchemaStringFormat.Date:
+                    if (!Regex.IsMatch(str, @"^(\d{4})-(\d{2})-(\d{2})$"))
```

Comment:
>(\d{2})-(\d{2}) [](start = 55, length = 15)

is this too direct.?

Reply:
See https://github.com/dotnet/docfx/pull/4914/files#diff-85968dd7fbc00463b422b61d04bdf8f8R13, `DateTime.Parse` is too permissive with respect to JSON schema spec.

Reply:
I mean for date like '1990-13-99'